packages/video_player/README.md

```diff
@@ -59,29 +69,32 @@ class _MyHomePageState extends State<MyHomePage> {
           });
         }
       })
-      ..initialize();
+      ..initialize().then((_) {
+        print("Initialized ${_controller.value.initialized}");
```

Comment:
Did you intend to leave this `print` call here?

Reply:
No!